### ISSUE TITLE
Enhance wins JavaScript to assign alt text to LinkedIn icon

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4010]
### Overview
To comply with Web Content Accessibility Guidelines (WCAG), we need to ensure that image alt text on the Wins page is meaningful and appropriate for each icon.  This issue addresses the need for alt text on the LinkedIn icon in the Wins cards, which are generated using JavaScript. 

### Action Items

- [x] Understand the role of these components of the wins page: 
  - [x] The main Wins web page: [wins.html](/hackforla/website/blob/gh-pages/pages/wins/wins.html), which uses Liquid to include the template for each win card:  [wins-card-template.html](/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
  - [x] The JavaScript code [wins.js](hackforla/website/blob/gh-pages/assets/js/wins.js) running in the Wins web page. 
- [x] In  [wins-card-template.html](/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html), find the img element with class 'linkedin-icon', and add an empty alt attribute (`alt=''`). 
- [x] In [wins js](/hackforla/website/blob/gh-pages/assets/js/wins.js), search for the text '.linkedin-icon' in order to locate the line of code which sets the "src" attribute for the GitHub icon.  Add code to set the "alt" attribute of the icon equal to "GitHub profile for `name`" where `name` is the name of the contributor.  For example, the alt text for the image representing the link to Mary Smith's LinkedIn page should be "GitHub profile for Mary Smith".  
- [x] A change will also be required in wins.js in the function updateOverlay to set the alt text in the Overlay (when the user clicks "See More" for wide screens).  
- [x] View the wins page to test the code at mobile, tablet and full-screen widths. Also also follow the "See More" page to check the alt tag on the modal overlay.   

### Resources/Instructions

- [Wins webpage](https://hackforla.org/wins/)
- [wins.html](/hackforla/website/blob/gh-pages/pages/wins/wins.html)
- [wins js](/hackforla/website/blob/gh-pages/assets/js/wins.js)
- [wins-card-template.html](/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
- This issue was created as a follow-up to #2569.  See that issue for accessibility resources.

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe let's talk about the instructions at the next meeting.  I have a question about this part

> - [ ] In [wins.js](https://github.com/hackforla/website/issues/hackforla/website/blob/gh-pages/assets/js/wins.js), search for the text '.linkedin-icon' in order to locate the line of code which sets the "src" attribute for the LinkedIn icon. Add code to set the "alt" attribute of the icon equal to "Linked in profile for name" where name is the name of the contributor.

**Comment by ExperimentsInHonesty** [comment]
Got clarification, that the change will result in having alt text on Bonnie's win card that says "Linked in profile for Bonnie Wolfe"

**Comment by ExperimentsInHonesty** [comment]
- see note on #4088

**Comment by jdingeman** [comment]
Removing the dependency and its section. Marking as ready for prioritization.

**Comment by roslynwythe** [comment]
Availability: 3/19 7 - 10 pm
ETA EOD 3/21
